runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -204,6 +226,19 @@ public void onGameTick(GameTick tick)
 			return;
 		}
 
+		for (DriftNet net : NETS)
+		{
+			// untag fish which are next to a net that is not accepting fish right now
+			// or was not accepting fish previous tick (When a fish shoal is 2 tiles adjacent
+			// to a drift net and is moving to a net that is just being setup it will be denied
+			// even though the net is currently in the CATCHING status)
+			if (net.isNotAcceptingFish() || net.wasNotAcceptingFishPrevTick())
+			{
+				taggedFish.entrySet().removeIf(entry -> checkFishOnTiles(entry.getKey(), net.getAdjacentTiles()));
+			}
+			net.setPrevTickStatus();
```

Comment:
This will be the same as the current status unless there has been a change this tick, so it might be worth considering setting this in onVarbitChanged instead? not sure.

Reply:
Don't really know how that would work. We would still need to make sure that the next tick after a VarbitChanged, the prevTickStatus would be the same as the current status.